api.raml

```diff
@@ -157,6 +157,28 @@ uses:
             body:
               application/json: lib.DifferentialAbundanceStatsResponse
 
+  /correlationassaymetadata:
```

Comment:
im ok w this, but id prefer if we had a clean and easy way to do correlation/assaymetadata instead

Reply:
I tried a bit but i think that would require more help from infra, as it's not straightforward to me at least. I don't mind revisiting the idea but i think the work is outside the scope of this PR.

Reply:
https://github.com/VEuPathDB/service-eda-compute/issues/71

Reply:
yea i played this game in the data service once and it wasnt straightforward there either, which is a shame. :(